treelite.sh

```diff
@@ -0,0 +1,45 @@
+package: treelite
+version: "%(tag_basename)s"
+tag: "2a12742269c1d3de553d9a12ff36bc2a5d874239"
+source: https://github.com/dmlc/treelite
+requires:
+  - "GCC-Toolchain:(?!osx)"
+build_requires:
+  - CMake
+  - "Xcode:(osx.*)"
+incremental_recipe: |
+  make ${JOBS:+-j$JOBS} install
+  mkdir -p $INSTALLROOT/etc/modulefiles && rsync -a --delete etc/modulefiles/ $INSTALLROOT/etc/modulefiles
+---
+#!/bin/bash -e
+
+pushd $SOURCEDIR
+git submodule update --init --recursive
```

Comment:
Please `rsync` to the current dir, and then run the submodule command. Please also indent the `pushd...popd` "block" - yes, it's not a block, and Bash's not a proper language, but still :-)